README.md

```diff
@@ -7,8 +7,10 @@ wolfSSL's Embeddable SSH Server
 dependencies
 ------------
 
-[wolfSSH](https://www.wolfssl.com/wolfssh/) is dependent on [wolfCrypt](https://www.wolfssl.com/download/).
-The simplest configuration of wolfSSL required for wolfSSH is the default build.
+[wolfSSH](https://www.wolfssl.com/wolfssh/) is dependent on
+[wolfCrypt](https://www.wolfssl.com/download/).
```

Comment:
this is somewhat confusing as wolfcrypt isn't a package you can download.  You might want to say "which is part of wolfSSL" or just replace wolfCrypt with wolfSSL.

Reply:
Updated.